lib/mayaUsd/fileio/utils/meshWriteUtils.cpp

```diff
@@ -1408,7 +1408,8 @@ MStatus UsdMayaMeshWriteUtils::exportComponentTags(UsdGeomMesh& primSchema, MObj
     MPlug outShp = depNodeFn.findPlug("outMesh", &status);
     CHECK_MSTATUS_AND_RETURN_IT(status);
 
-    MDataHandle geomDataHandle = outShp.asMDataHandle();
+    auto        outShpHolder = UsdMayaUtil::GetPlugDataHandle(outShp);
+    MDataHandle geomDataHandle = outShpHolder->GetDataHandle();
     MObject     geomObj = geomDataHandle.data();
```

Comment:
Is this making a needless copy to store in geomDataHandle?
```suggestion
    MObject     geomObj = outShpHolder->GetDataHandle().data();
```